src/app/workspace/resource/representation/add-region-form/add-region-form.component.html

```diff
@@ -0,0 +1,42 @@
+<form [formGroup]="regionForm">
+    <mat-form-field>
+        <mat-label>Comment</mat-label>
+        <input matInput formControlName="comment">
+    </mat-form-field>
+    <mat-form-field>
+        <mat-label>Label</mat-label>
```

Comment:
Add an asterisk to the label, so the user knows that a value is required.

Reply:
Done